### ISSUE TITLE
Duplicated Select when using browser-default. Alpha 3

### ISSUE THREAD

**Issue by xsama** [Dogfalo/materialize#5556]
Duplicated Select when using browser-default. Alpha 3
 

## Current Behavior
like we can see in  [next.materializecss.com/select.html](http://next.materializecss.com/select.html) if we use   `<select class="browser-default">`.

![borrame](https://user-images.githubusercontent.com/2250307/34934365-dcf73a8c-f99f-11e7-9db9-1664835a39e3.png)

## Possible Solution
i have no idea, sorry i'm new on this and i don't have many time, i'm just not initializing it by the moment.
... but maybe this can help u.

New materialize.js alpha3
```
 key: "init",
      value: function init(els, options) {
        return _get(Range.__proto__ || Object.getPrototypeOf(Range), "init", this).call(this, this, els, options);
      }

```
maybe must look for 'browser-default' like the old version. 
Old materialize.js alpha2 i think

```
 static init($els, options) {
      let arr = [];
      $els.each(function() {
        if (!$(this).hasClass('browser-default')) {
          arr.push(new Select(this, options));
        }
      });
      return arr;
    }
```

## Steps to Reproduce (for bugs)
```
<div class="col s12">
    <br>
    <p>You can add the class  browser-default o get the browser default.</p>
</div>
<div class="col s12 m6">
    <label>Browser Select</label>
    <select class="browser-default">
        <option value="" disabled selected>Choose your option</option>
        <option value="1">Option 1</option>
        <option value="2">Option 2</option>
        <option value="3">Option 3</option>
    </select>
</div>

```

## Thanks a lot for your hard work.
i love materialize, my work is a lot easier, i hope i can donate in my next payment.

**Comment by ray007** [comment]
I've made a local css update that works for me, see my comment in #5294

**Comment by acburst** [comment]
Fixed in 4bbcb17d

**Comment by ray007** [comment]
Better.
But it falls apart if wrapped in a `.input-field`, and misses the top- and bottom-margins if not.
